ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables/Basics.hs

```diff
@@ -79,15 +121,15 @@ deriving newtype instance NoThunks (mk (Key l) (Value l))
 -- | Each @LedgerState@ instance will have the notion of a @Key@ for the tables.
 -- For instance, if we only pulled out only the UTxO set from the ledger state,
 -- this type would be @TxIn@. See
--- "Ouroboros.Consensus.HardFork.Combinator.Ledger.CanonicalTxIn".
+-- "Ouroboros.Consensus.HardFork.Combinator.Ledger".
 type Key :: LedgerStateKind -> Type
-type family Key l
+type family Key l -- TODO: rename to TxIn
 
 -- | Each @LedgerState@ instance will have the notion of a @Value@ for the
 -- tables. For instance, if we only pulled out only the UTxO set from the ledger
 -- state, this type would be @TxOut@ or @NS TxOut@.
 type Value :: LedgerStateKind -> Type
-type family Value l
+type family Value l -- TODO: rename to TxOut
```

Comment:
Not a very interesting change, but it is entirely separate from what I've been doing in this PR so let's do it in a new PR

Reply:
And actually, is this something we want to change? Are ledgertables UTxO-specific, or would we reuse the same type for other tables like stake-related tables?